api-docs-slate/source/includes/_wallet.md

```diff
@@ -1,4 +1,38 @@
 # Wallet
+## The Wallet Client
+The best way to interact with the wallet API is with the bwallet-cli in the `bclient`
+[package](https://www.npmjs.com/package/bclient). Installing globally with
+`npm i -g bclient` gives you access to the cli. You can also install locally
+to your project.
+
+Note that when using it in your own program, you will need to explicitly
+pass in a port option. The easiest way to do this is with `bcoin.Network`.
+
+You can create a client for a specific wallet (and be compatible with the old
+api) with the `wallet` method on `WalletClient` class.
+
+```shell--curl
+```
+
+```shell--cli
+npm i -g bclient && bwallet-cli
+```
+
+```javascript
+const { WalletClient } = require('bclient');
+const { Network } = require('bcoin');
+const network = Network.get('testnet');
+
+const walletClient = new WalletClient({
+  port: network.walletPort,
+  network: network.type
+});
+
+const id = 'primary'; // or 'foo' or whatever
```

Comment:
Maybe change comment to something less ambiguous?  Something like
`// name of wallet` or `// any wallet name` ?